rest-api/data/sites.csv

```diff
@@ -1,191 +1,190 @@
+SEEC_MOREHOUSE,hpo-site-morehousehowellmillsrd,Morehouse School of Medicine - Howell Mills Rd.,HPO Morehouse Howell Mills Rd.,7036508,,,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,"Morehouse Healthcare, Howell Mill Office",1800 Howell Mill Road,Suites 275 and 550 ,Atlanta,GA,30318,,,
+SEEC_MOREHOUSE,hpo-site-morehousegrady,Morehouse School of Medicine - Grady,HPO Morehouse Grady,7036509,,,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,Morehouse School of Medicine,80 Jesse Hill Drive,,Atlanta,GA,30303,,,
+SEEC_MIAMI,hpo-site-umiamibrb,University of Miami - Biomedical Research Bldg.,HPO University of Miami BRB,7036535,,,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,University of Miami ,1501 NW 10th Avenue,BRB Rm 343,Miami,FL,33136,,,
+SEEC_MIAMI,hpo-site-umiamipac,University of Miami - Professional Arts Center ,HPO University of Miami PAC,7036535,,,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,"University of Miami, Professional Arts Center (PAC Building) ",1150 NW 14th Street,Suite 411,Miami,FL,33136,,,
+SEEC_MIAMI,hpo-site-umiamimeu,University of Miami - Mobile Enrollment Unit,HPO University of Miami MEU,7036535,,,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,,,,,,,,,
+SOUTHERN_UAB,hpo-site-uabkirklin,University of Alabama Birmingham (UAB) Kirklin Clinic,HPO UAB Kirklin,7036507,3/19/18,,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,,,,,,,,,,"aacemgil@uabmc.edu, coden@uabmc.edu",
+SOUTHERN_UAB,hpo-site-uabmedicaltowers,UAB Medical Towers,HPO UAB Medical Towers,7036507,3/19/18,,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,,,,,,,,,,"aacemgil@uabmc.edu, coden@uabmc.edu",
+SOUTHERN_UASB,hpo-site-usafamilymedical,University of South Alabama Family Medical Center (USA) ,HPO USA Family Medical,,,,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,,,,,,,,,
+SOUTHERN_UASB,hpo-site-usaphysiciansgroup,University of South Alabama Physicians Group (USAPG),HPO USA Physicians Group,,,,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,,,,,,,,,
+SOUTHERN_UASB,hpo-site-usastantonroad,University of
```

Comment:
Make sure that recent changes in sites.csv are reflected here.

Reply:
done